faker/providers/color/color.py

```diff
@@ -144,37 +144,64 @@ def __init__(self, generator: Optional["Generator"] = None, seed: Optional[Hasha
             self.seed = seed if seed else random.randint(0, sys.maxsize)
             self.random = random.Random(self.seed)
 
-        for color_name, color_attrs in self.colormap.items():
-            lower_bounds: Sequence[Tuple[int, int]] = color_attrs["lower_bounds"]
-            s_min, b_max = lower_bounds[0]
-            s_max, b_min = lower_bounds[-1]
-
-            self.colormap[color_name]["saturation_range"] = [(s_min, s_max)]
-            self.colormap[color_name]["brightness_range"] = [(b_min, b_max)]
-
     def generate(
         self,
         hue: Optional[HueType] = None,
         luminosity: Optional[str] = None,
         color_format: str = "hex",
```

Comment:
can we use the [`Literal` type hint](https://docs.python.org/3.9/library/typing.html#typing.Literal) here?

Reply:
good idea, done.